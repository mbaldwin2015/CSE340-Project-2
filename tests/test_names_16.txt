xyz, rst, tmp;

hello {
  public: xyz, rst;
  private: tmp, abc;
  
  world1 {
    xyz = rst;
    tmp = abc;
  }
  world2 {
    public: xyz, tmp;

    xyz = rst;
    tmp = abc;
  }
}
