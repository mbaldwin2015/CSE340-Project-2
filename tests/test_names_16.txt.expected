hello.xyz = hello.rst
::tmp = ?.abc
world2.xyz = hello.rst
world2.tmp = ?.abc
